### DISCUSSION TITLE
第2节课作业(17班)

### DISCUSSION THREAD

**Discussion by Jianfeng777** [InternLM/Tutorial#110] [General]
请将本次的作业也以链接的形式上传，并同时登记在表格当中

**Comment by fine-lab** [comment]
第二节课作业
https://blog.csdn.net/SmallZCan_t/article/details/135447013?spm=1001.2014.3001.5502

**Comment by Jianfeng777** [comment]
第二节课作业
https://blog.csdn.net/jianfengkexin/article/details/135409765?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135409765%22%2C%22source%22%3A%22jianfengkexin%22%7D

**Comment by Alisa742** [comment]
第二次课作业：https://blog.csdn.net/weixin_53222965/article/details/135450446?spm=1001.2014.3001.5501

**Comment by zcg-joker** [comment]
[第二节课作业](https://blog.csdn.net/m0_55764641/article/details/135452507)

**Comment by AxrLH** [comment]
第二节课作业：https://blog.csdn.net/Axr_Samuel/article/details/135456433?spm=1001.2014.3001.5501

**Comment by langlibai66** [comment]
第二次课程作业：http://t.csdnimg.cn/40g74

**Comment by EthanHan1999** [comment]
第二节课作业：https://blog.csdn.net/weixin_44015617/article/details/135483576?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135483576%22%2C%22source%22%3A%22weixin_44015617%22%7D

**Comment by xaiojianjianbujiandan** [comment]
第二节课作业：https://blog.csdn.net/weixin_50256876/article/details/135482760

**Comment by popcornGit** [comment]
第二节课作业
https://blog.csdn.net/weixin_38771145/article/details/135489662

**Comment by JACK626** [comment]
第二节课作业 https://www.yuque.com/xiaotuanyue/mdr8d5/hwvcg4fq41elo7ec?singleDoc# 《第二节课笔记+作业》

**Comment by futtttttttttttt** [comment]
https://zhuanlan.zhihu.com/p/676932737

**Comment by StarDuster** [comment]
第二次作业 https://gist.github.com/StarDuster/5181eed8afc2dafc6544612b1af6d432

**Comment by l241025097** [comment]
第二节作业
https://github.com/InternLM/tutorial/files/13856101/02.Demo.pdf
